### PR TITLE
fix: incorrect HTTP request body with the Golang code generator

### PR DESCRIPTION
**Pull request by Agility6**
incorrect HTTP request body with the Golang code generator #364

### CODE REVIEW

File: pkg/generator/data/main.go.tpl [hunk 1]
```diff
@@ -13,7 +16,7 @@ func main() {
 	{{- end}}
 	body := strings.NewReader(data.Encode())
 	{{- else}}
-	body := bytes.NewBufferString("{{.Request.Body.String}}")
+	body := bytes.NewBufferString("{{.Request.Body.String | safeString}}")
```
Comment (SLOWDOWNO) [inline pkg/generator/data/main.go.tpl:19]: The syntax I feel can work is: ```body := bytes.NewBufferString(`{{.Request.Body.String | safeString}}`)```

Comment (Agility6) [reply pkg/generator/data/main.go.tpl:19]: Yes, it solves the problem of escaping

Comment (Agility6) [reply pkg/generator/data/main.go.tpl:19]: This PR is in WIP status and I will continue to improve it.

File: pkg/generator/testdata/expected_go_body_request_code.txt [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+package main
+
+import (
+	"bytes"
+	"io"
+	"net/http"
+)
+
+func main() {
+	body := bytes.NewBufferString("key: "value"")
```
Comment (LinuxSuRen) [inline pkg/generator/testdata/expected_go_body_request_code.txt:22]: image

**Review by LinuxSuRen — CHANGES_REQUESTED** [review]

**Review by LinuxSuRen — APPROVED** [review]

**Review by SLOWDOWNO — APPROVED** [review]

**Comment by LinuxSuRen** [comment]
Thanks @Agility6 and @SLOWDOWNO , I'm going to merge this.

**Pull request merged**